contentcuration/contentcuration/views.py

```diff
@@ -262,10 +263,11 @@ def duplicate_nodes(request):
             new_nodes = []
 
             with transaction.atomic():
-                for node_data in nodes:
-                    new_node = _duplicate_node(node_data['id'], sort_order=sort_order, parent=target_parent, channel_id=channel_id)
-                    new_nodes.append(new_node.pk)
-                    sort_order+=1
+                with ContentNode.objects.disable_mptt_updates():
```

Comment:
Only updating one tree (not sure what you're saying the implications of that are, though), and `delay` vs `disable` doesn't affect performance, but I figured it better clarifies what's going on.